README.rst

```diff
@@ -26,14 +46,16 @@ normally used in B physics is also provided.
     minuit.migrad()
     unbinned_likelihood.draw(minuit)
 
+Documentation and Tutorial
+--------------------------
 
-* `MIT <http://opensource.org/licenses/MIT>`_ license (open source)
-* `Documentation <http://probfit.readthedocs.org/>`_
+*`Documentation <http://probfit.readthedocs.org/>`_
```

Comment:
A space is now missing here, it won't render correctly.

Reply:
Good points, thanks!